closed/adds/jdk/src/windows/native/jdk/crypto/jniprovider/NativeCrypto_md.c

```diff
@@ -61,3 +82,85 @@ void * find_crypto_symbol(void *handle, const char *symname) {
     return symptr;
 }
 
+/*
+ * Find path to JRE based on .exe's location
+ */
+jboolean
+GetJREPath(char *path, jint pathsize)
```

Comment:
This and the following functions look like copies from `jdk/src/windows/bin/java_md.c`; the copyright notice needs to reflect that.

Reply:
Yes, they are. I tried using them from that file to avoid copying them but was not able to do so. 
Will update the file to include the relevant copyright notices.

Reply:
Why here and not at the top of the file like other files with contributions from IBM?

Reply:
Probabaly somantics, but this was a file written by IBM that had some methods from Oracle added, and not an Oracle file that had IBM methods added. I did check with the lawyer here in Hursley and they said add the copyright prior to the block of code that was copied from the Oracle file.

Reply:
The top of the file is 'prior' to the Oracle code that was added. I won't push it, but it would eliminate the copyright checker issue.